dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeConfig.java

```diff
@@ -82,6 +82,10 @@
 	// Minimum allowed time to check for expiration
 	private Long renewCertMin;
 	
+	private Integer httpConnectTimeout = AcmeConstants.HTTP_CONNECT_TIMEOUT_DEFAULT;
```

Comment:
If you change the types in the metatype to `ibm:type="duration"` then these will need to be Long.

Reply:
Changed to duration